### ISSUE TITLE
Select text bad misaligned in Firefox with alpha 5

### ISSUE THREAD

**Issue by fredgate** [twbs/bootstrap#21096]
The text in the input selects is misaligned vertically. The bottom of the text is clipped; the text is not centered vertically and is too down.
I have the problem since I upgraded to **alpha 5**; before with the aplha 2 version, I had not the problem.
This occurs in **Firefox** 49.0.2; in Internet Explorer 11, it seems correct.

Here is a jsfiddle showing the problem :
  [https://jsfiddle.net/hdakuemq/](https://jsfiddle.net/hdakuemq/)

**Comment by fredgate** [comment]
May be that it is this rule that produces the problem :
```
select.form-control:not([size]):not([multiple]) {
  height: calc(2.5rem - 2px);
}
```
When I deactivate it, the padding is correct and the display good; but the select is too high.

**Comment by nextgenthemes** [comment]
Firefox 49.0.2 on ubuntu 16.04 I see no clipped text. A screenshot woudl be helpfull I cant see any issue.

**Comment by fredgate** [comment]
Here is a screenshot showing the problem with 49.0.2 on Windows 7 64 bits.

![temp](https://cloud.githubusercontent.com/assets/3648269/20186826/e026f480-a771-11e6-9319-c83a7b7e91c4.png)

**Comment by nextgenthemes** [comment]
OK I don't see that in Ubuntu.

**Comment by kewagi** [comment]
Firefox 50.0 on OS X still has this problem. Chrome 54.0.2840.98 displays this without clipping.

`<li class="form-group">
	<label for="id_project_type_education_context">
		Diplomarbeit / Dissertation
	</label>
	<select class="form-control">
		<option>Dissertation</option>
		<option>Diplomarbeit</option>
	</select>
</li>`

![select error](https://cloud.githubusercontent.com/assets/84195/20432145/1cb61458-ad9e-11e6-9e53-a6a7880be4b3.png)

**Comment by kewagi** [comment]
Firefox 50.0 on OS X still has this problem. Chrome 54.0.2840.98 displays this without clipping.

```
<li class="form-group">  
	<label for="id_project_type_education_context">  
		Diplomarbeit / Dissertation  
	</label>  
	<select class="form-control">  
		<option>Dissertation</option>  
		<option>Diplomarbeit</option>  
	</select>  
</li>
```

![select error](https://cloud.githubusercontent.com/assets/84195/20432145/1cb61458-ad9e-11e6-9e53-a6a7880be4b3.png)

**Comment by bardiharborow** [comment]
Cannot reproduce on Firefox 51.0a2. Possible browser bug.

![Screenshot](https://cloud.githubusercontent.com/assets/1073681/20467282/b04def1a-afd8-11e6-9dbb-712f7cd74339.png)

**Comment by kewagi** [comment]
Problem still there with Firefox 51.0b1 under macOS 10.12.1. Might be a Browser+OS problem, or potentially related to custom font size?

**Comment by mdo** [comment]
Dupe of #20977, fixed with #21171.
